game/src/starter/Game.java

```diff
@@ -225,6 +228,11 @@ private void setupDSLInput() {
 
     public static void main(String[] args) {
         // start the game
+        try {
+            Configuration.loadAndGetConfiguration("dungeon_config.json", KeyboardConfig.class);
+        } catch (IOException e) {
```

Comment:
sollten wir irgendwie eine default config mitgeben?

Reply:
Die DefaultWerte werden in den jeweiligen ConfigMaps angegeben. Eine Standardkonfiguration wird generiert, wenn das erste mal gestartet wird und keine Konfigurationsdatei vorhanden ist.

Reply:
Hoffe ist jetzt besser verständlich mit der Docu :)